### PR TITLE
update README to clarify the current status of this repo

### PR DESCRIPTION
**Pull request by AkihiroSuda**
Let's have this update in the master and `release/1.4` branch.

### CODE REVIEW

**Comment by AkihiroSuda** [comment]
@dmcgowan PTAL

File: README.md [hunk 1]
```diff
@@ -1,17 +1,22 @@
+# Moved to [`github.com/containerd/containerd/pkg/cri`](https://github.com/containerd/containerd/tree/master/pkg/cri)
+
+On October 7, 2020, the contents of this repo were moved to [`pkg/cri` package](https://github.com/containerd/containerd/tree/master/pkg/cri)
+in [the main containerd repo](https://github.com/containerd/containerd).
+
+**Pull requests are no longer accepted in the master branch of this repo.**
+PRs for `release/1.3` and `release/1.4` branches are still accepted, but the commits must be merged into the
+main repo first, and then backported to this repo.
```
Comment (mikebrow) [inline README.md:8]: s/accepted,/, e.g. for service updates./
s/but the commits must be merged into the main repo/However, the master branch for containerd/cri integration work is now located in the containerd/containerd repository, and as such new commits should be merged there/

**Review by mikebrow — COMMENTED** [review]
see suggested edits

File: README.md (hunk 1 above)
Comment (AkihiroSuda) [reply README.md:8]: What do you mean by "service updates"?

Comment (mikebrow) [reply README.md:8]: I mean the 1.3 and 1.4 branches are service branches.. 
By service updates I mean updates to fix bugs, improve performance, and resolve security issues.

File: README.md [hunk 2]
```diff
@@ -1,17 +1,22 @@
+# Moved to [`github.com/containerd/containerd/pkg/cri`](https://github.com/containerd/containerd/tree/master/pkg/cri)
+
+On October 7, 2020, the contents of this repo were moved to [`pkg/cri` package](https://github.com/containerd/containerd/tree/master/pkg/cri)
```
[the remark this answers is not in the document]
Comment (AkihiroSuda) [reply README.md:3]: applied

File: README.md (hunk 1 above)
Comment (AkihiroSuda) [reply README.md]: Updated. I think we can simply call it "bug-fix PRs". PTAL.

**Review by mikebrow — APPROVED** [review]
LGTM

**Comment by MHBauer** [comment]
https://github.com/kubernetes/kubernetes/pull/95613 is causing some of your pull-cri-containerd-node-e2e failure.

**Comment by AkihiroSuda** [comment]
The code is no longer maintained in this repo, so let's merge this as-is

**Review by fuweid — APPROVED** [review]
LGTM

**Comment by fuweid** [comment]
README.md change doesn't break code. But it needs to tell what happens right now. Merging....

**Comment by AkihiroSuda** [comment]
cherry-pick: https://github.com/containerd/cri/pull/1599

**Pull request merged**